src/core/kmod-setup.c

```diff
@@ -120,6 +121,9 @@ int kmod_setup(void) {
 
                 /* dmi-sysfs is needed to import credentials from it super early */
                 { "dmi-sysfs", "/sys/firmware/dmi/entries", false, false,  NULL   },
+
+                /* Make sure the tpm module is available so early boot services have access to the tpm. */
+                { "tpm", "/sys/class/tpmrm", false, false, efi_has_tpm2 },
```

Comment:
please align the fields like in the rest of the table.

(yeah, some lines broke the alignments already, but please feel free to fix that, at the same time.)

please say in the comment that this is not about access to the tpm, but about the tpm *subsystem*. i.e. you could otherwise assume this line would give access to the tpm chip, but it really doesn't, it only powers up the tpm subsystem without any drivers after all.

please conditionalize the whole line via `#if HAVE_TPM2`